data/data_pipe.py

```diff
@@ -108,6 +116,37 @@ def load_mx_rec(rec_path):
             label_path.mkdir()
         img.save(label_path/'{}.jpg'.format(idx), quality=95)
 
+
+class CarrayLoader:
+    def __init__(self, carray, conf):
+        self.carray = carray
+        self.batch_size = conf.batch_size
+
+    def __iter__(self):
+        self.idx = 0
+        return self
+
+    def __next__(self):
+        if self.idx + self.batch_size <= len(self.carray):
+            output = torch.tensor(self.carray[self.idx: self.idx + self.batch_size])
+        elif self.idx < len(self.carray):
+            output = torch.tensor(self.carray[self.idx:])
+        else:
+            raise StopIteration
+        self.idx += self.batch_size
+        return output
+
+
+def loader_from_carray(carray: np.ndarray, conf, strategy='iterable'):
```

Comment:
When will we want to use the strategy='iterable'?

Reply:
When the whole dataset is small enough to be put in the memory